### ISSUE TITLE
Fix news template

### ISSUE THREAD

**Issue by henrikssn** [fsek/web#185]
````
I, [2015-05-13T16:46:15.451426 #30237]  INFO -- : Processing by StaticPagesController#index as HTML
I, [2015-05-13T16:46:15.499552 #30237]  INFO -- :   Rendered news/_news.html.erb (2.7ms)
I, [2015-05-13T16:46:15.499693 #30237]  INFO -- :   Rendered static_pages/index.html.erb within layouts/application (4.5ms)
I, [2015-05-13T16:46:15.499947 #30237]  INFO -- : Completed 500 Internal Server Error in 7ms (ActiveRecord: 0.5ms)
F, [2015-05-13T16:46:15.501887 #30237] FATAL -- : 
ActionView::Template::Error (private method `print' called for nil:NilClass):
    3:       <h4><%=link_to(news.title,news)%></h4>
    4:       <%=simple_format(truncate(news.content,length: 140, separator: '', escape: false){ link_to "Läs mer", news })%>
    5:       <ul class="list-unstyled">
    6:         <li><i class="fa fa-user"> </i><%=link_to(news.user.print,news.user)%></li>
    7:         <li><i class="fa fa-calendar"></i> <%=news.created_at%></li>
    8:       </ul>
    9:     </div>
  app/views/news/_news.html.erb:6:in `_app_views_news__news_html_erb__1250118806894294363_70356068145700'
  app/views/static_pages/index.html.erb:19:in `_app_views_static_pages_index_html_erb___2070542433745366028_70356067180020'
````

I deleted line 6 above in #184. Fix the error and add it back.

**Comment by henrikssn** [comment]
I'll assign @davidwessman on this one as I think you know this code better than me :)

**Comment by davidwessman** [comment]
Will fix this.

**Comment by davidwessman** [comment]
How should we test this?
This should be seen by our feature tests, but only if there is News-content.
Any ideas? @fsek/admins

**Comment by henrikssn** [comment]
Maybe feed the test with (the) test data in a before_each function?

**Comment by davidwessman** [comment]
Just add 'News' or our whole tests_data rake-task?

**Comment by henrikssn** [comment]
The whole rake task if you can get it to work, otherwise maybe some "essentials" which we expect to be there :) I.e. news, static pages, some users, a council

**Comment by davidwessman** [comment]
Yepp, makes sense 😄

**Comment by davidwessman** [comment]
https://teamgaslight.com/blog/6-ways-to-remove-pain-from-feature-testing-in-ruby-on-rails
He recommends FactoryGirl.

**Comment by davidwessman** [comment]
@henrikssn 
Did we move News.profile to News.user?

**Comment by davidwessman** [comment]
Closed by #186.
